playbook/look-for-build-regression-statistics.md

```diff
@@ -1,3 +1,45 @@
-# :construction: Look for build regression statistics :construction:
+# Look for build regression statistics 
+
+This play will help you gather statistics about build regressions in a job.
+
+> **Note**
+> Unlike other plays, this one uses different set of tools:
+
+## Steps
+
+Using testdb you can get historical data between builds in a job. This way, you can compare a passing and a failing build.
+
+First, move to the `testdb` folder.
+
+Then, set the domain and job you want to check via environment variables, and fetch the logs:
+
+```bash
+export domain=[domain]
+export job=[job]
+
+./fetch.rb -d $domain -j $job -c [build-count] buildlog
```

Comment:
Please add a note so users know that this script will download `build-count` amount of logs to their local system